config/psyclone.cfg

```diff
@@ -33,6 +33,7 @@
 # -----------------------------------------------------------------------------
 # Author: A. R. Porter, STFC Daresbury Laboratory
 # Modified by: R. W. Ford, STFC Daresbury Laboratory
+#              J. Henrichs, Bureau of Meteorology
```

Comment:
Please update the copyright date at the top too.

Reply:
Done.